client/layout/html-is-iframe-classname.tsx

```diff
@@ -0,0 +1,38 @@
+/**
+ * External dependencies
+ */
+import { useEffect, memo } from 'react';
+import { useSelector } from 'react-redux';
+import debugFactory from 'debug';
+
+/**
+ * Internal dependencies
+ */
+import isIframeForHtmlElement from 'state/selectors/is-iframe-for-html-element';
+
+const debug = debugFactory( 'calypso:layout:html-is-iframe-classname' );
+
+const HtmlIsIframeClassname = memo( () => {
+	const isIframe = useSelector( isIframeForHtmlElement );
+
+	useEffect( () => {
+		const htmlNode = document.querySelector( 'html' );
+
+		if ( ! htmlNode ) {
+			debug( 'no html node' );
+			return;
+		}
+
+		if ( isIframe ) {
+			debug( 'adding is-iframe' );
+			htmlNode.classList.add( 'is-iframe' );
+		} else {
+			debug( 'removing is-iframe' );
+			htmlNode.classList.remove( 'is-iframe' );
+		}
+	} );
```

Comment:
There's one more important thing that should go into the `useEffect` here: return an "unmount" callback that removes the `is-iframe` classname when the `HtmlIsIframeClassname` is unmounted:
```js
return () => htmlNode.classList.remove( 'is-iframe' )
```
If you don't do this, bugs like the one fixed in #36424 will happen.